frontend/packages/vsphere-plugin/src/components/VSphereConnectionForm.tsx

```diff
@@ -0,0 +1,238 @@
+    setDatacenter,
+    setDefaultdatastore,
+    setFolder,
+  } = useConnectionFormContext();
+
+  React.useEffect(() => {
+    if (vcenterRef && vcenterRef.current) {
+      (vcenterRef.current as HTMLInputElement).focus();
+    }
+  }, []);
+
+  // initial load
+  React.useEffect(() => {
+    const doItAsync = async () => {
+      if (isLoaded) {
+        return;
+      }
+      if (!cloudProviderConfig) {
+        return;
+      }
+
+      setIsLoaded(true);
+      await initialLoad(
+        {
+          setDirty,
+          setVcenter,
+          setUsername,
+          setPassword,
+          setDatacenter,
+          setDefaultdatastore,
+          setFolder,
+        },
+        SecretModel,
+        cloudProviderConfig,
+      );
+    };
+
+    doItAsync();
+  }, [
+    SecretModel,
+    cloudProviderConfig,
+    isLoaded,
+    setDatacenter,
+    setDefaultdatastore,
+    setDirty,
+    setFolder,
+    setPassword,
+    setUsername,
+    setVcenter,
+  ]);
+
+  const folderHelperText = (
+    <Trans i18nKey="vsphere-plugin~vsphere-connection-form-folderhelp-one">
+      Provide <b>datacenter</b> folder which contains VMs of the cluster, example: /
+      <span className="vsphere-connection-form-helper__datacenter">{{ datacenter }}</span>/<b>vm</b>
+      /<b>[MY_VMS_TOP_FOLDER]</b>.
+    </Trans>
+  );
+
+  const datacenterHelperText = (
+    <>
+      <Trans i18nKey="vsphere-plugin~vsphere-connection-form-datacenterhelp-one">
+        The name of an existing datacenter in the vSphere which the virtual machines backing this
+        cluster are in.
+      </Trans>
+      <br />
+      <strong>
+        <Trans i18nKey="vsphere-plugin~vsphere-connection-form-datacenterhelp-two">
+          Please mind, that changing this value will break existing PersistentVolumes, if there are
+          already any.
+        </Trans>
+      </strong>
+    </>
+  );
+
+  const datastoreHelperText = (
+    <>
+      <Trans i18nKey="vsphere-plugin~vsphere-connection-form-datastorehelp-one">
+        The name of an existing datastore in the datacenter where the persistent volumes will be
+        stored.
+      </Trans>
+      <br />
+      <strong>
+        <Trans i18nKey="vsphere-plugin~vsphere-connection-form-datastorehelp-two">
+          Please mind, that changing this value will break existing PersistentVolumes, if there are
+          already any.
+        </Trans>
+      </strong>
+      .
+    </>
+  );
+
+  return (
+    <Form id={formId}>
+      <FormGroup
+        label={t('vsphere-plugin~vCenter')}
+        labelIcon={
+          <PopoverHelpButton
+            content={
+              <>
+                {t(
+                  "vsphere-plugin~Enter the network address the vCenter is running on. It can either be a domain name or IP. If you're unsure, you can try to determine the value from the vSphere Web Client address. Example: ",
+                )}
+                <ul>
+                  <li> https://[your_vCenter_address]/ui</li>
```

Comment:
or maybe the space is there on purpose ?
```suggestion
                  <li>https://[your_vCenter_address]/ui</li>
```

Reply:
done